lib/prefab/feature_flag_client.rb

```diff
@@ -2,53 +2,32 @@
 
 module Prefab
   class FeatureFlagClient
-    include Prefab::ConfigHelper
-    MAX_32_FLOAT = 4_294_967_294.0
-
     def initialize(base_client)
       @base_client = base_client
     end
 
-    def upsert(feature_name, feature_obj)
-      @base_client.config_client.upsert(feature_name, Prefab::ConfigValue.new(feature_flag: feature_obj))
-    end
-
     def feature_is_on?(feature_name)
       feature_is_on_for?(feature_name, nil)
     end
 
     def feature_is_on_for?(feature_name, lookup_key, attributes: {})
```

Comment:
🎱 should we rename them `properties` here too, like in the config client, while we're here?

Reply:
Good question. I have an open question for @jdwyah about standardizing this.